nginx.conf

```diff
@@ -1,104 +1,207 @@
+	# DNS API Verification Method
+	# Reference: https://github.com/acmesh-official/acme.sh/wiki/dnsapi
+	# /root/.acme.sh/acme.sh --force --issue --dns dns_cf -d ${hostname} -d "*.${hostname}"
+	# Deploy to nginx
+	# /root/.acme.sh/acme.sh --force --install-cert -d ${hostname} --key-file /etc/nginx/ssl/${hostname}/key.pem --fullchain-file /etc/nginx/ssl/${hostname}/fullchain.pem --ca-file /etc/nginx/ssl/${hostname}/chain.pem --reloadcmd "systemctl reload nginx"
+	# Change the below.
+	##
+
+	ssl_certificate /etc/nginx/ssl/plex.EXAMPLE.COM/fullchain.pem; 
+	ssl_key /etc/nginx/ssl/plex.EXAMPLE.COM/key.pem;
+	ssl_trusted_certificate /etc/nginx/ssl/plex.EXAMPLE.COM/fullchain.pem;
+	# Generate by doing openssl dhparam -out /etc/nginx/ssl/dhparam.pem 2048 
+	ssl_dhparam /etc/nginx/ssl/dhparam.pem; 
+	
+	# https://ssl-config.mozilla.org/#server=nginx&version=1.17.7&config=intermediate&openssl=1.1.1k&guideline=5.6
+	ssl_session_cache shared:MozSSL:10m;
+	ssl_session_timeout 1d;
+        ssl_protocols TLSv1.2 TLSv1.3;
+        ssl_ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384;
+        ssl_prefer_server_ciphers off;
 	ssl_stapling on;
 	ssl_stapling_verify on;
-	#For letsencrypt.org you can get your chain like this: https://esham.io/2016/01/ocsp-stapling
-	ssl_trusted_certificate /path/to/chain.pem;
-
-	#Reuse ssl sessions, avoids unnecessary handshakes
-	#Turning this on will increase performance, but at the cost of security. Read below before making a choice.
-	#https://github.com/mozilla/server-side-tls/issues/135
-	#https://wiki.mozilla.org/Security/Server_Side_TLS#TLS_tickets_.28RFC_5077.29
-	#ssl_session_tickets on;
-	ssl_session_tickets off;
-
-	#Use: openssl dhparam -out dhparam.pem 2048 - 4096 is better but for overhead reasons 2048 is enough for Plex.
-	ssl_dhparam /path/to/dhparam.pem;
-	ssl_ecdh_curve secp384r1;
-
-	#Will ensure https is always used by supported browsers which prevents any server-side http > https redirects, as the browser will internally correct any request to https.
-	#Recommended to submit to your domain to https://hstspreload.org as well.
-	#!WARNING! Only enable this if you intend to only serve Plex over https, until this rule expires in your browser it WONT BE POSSIBLE to access Plex via http, remove 'includeSubDomains;' if you only want it to effect your Plex (sub-)domain.
-	#This is disabled by default as it could cause issues with some playback devices it's advisable to test it with a small max-age and only enable if you don't encounter issues. (Haven't encountered any yet)
-	#add_header Strict-Transport-Security "max-age=63072000; includeSubDomains; preload" always;
-
-	#Plex has A LOT of javascript, xml and html. This helps a lot, but if it causes playback issues with devices turn it off. (Haven't encountered any yet)
-	gzip on;
```

Comment:
Did you encounter issues having gzip enabled, you accidentally removed it, or just didn't think it helps?